TensorFI/fiConfig.py

```diff
@@ -239,6 +239,69 @@ def __init__(self,fiParams):
 			self.skipCount = 0
 	# End of constructor
 
+	def updateInstance(self, op, confFile):
+		global fileName
+
+		#if op.value in Ops:
+		instances = confFile['Instances']
+
+		if instances is None:
+			instances = []
+
+		# Add the operation to the injectMap
+		self.totalInstance = self.totalInstance + 1
+		
+		if not op in self.opInstance:
+			self.opInstance[ op ] = 0
+		self.opInstance[ op ] = self.opInstance[ op ] + 1
```

Comment:
We ideally want to update the instance count of only those operations mentioned in the Ops field. All, if only "ALL" is mentioned. Otherwise, say, for "dynamicInstance" mode, it would inject in all operation instances once, even if we don't want to inject that operation (unless we remove it from the new config file manually).